virtual_labs/external/nexus/models.py

```diff
@@ -98,3 +98,18 @@ class NexusElasticSearchViewMapping(BaseModel):
 
 class NexusPermissions(NexusBase):
     permissions: List[str]
+
+
+class NexusSuiteProjects(BaseModel):
+    context: Annotated[str | list[str], Field(alias="@context")] = []
+    name: str
+    projects: str | list[str]
+
+
+ProjectView = TypedDict(
```

Comment:
@Dinika Minor nitpick for the future. The class syntax is slightly more convenient.

```py
class ProjectView(TypedDict):
     project: str
     viewID: str
```

Reply:
Done